src/app/tutorial.cljs

```diff
@@ -31,124 +31,124 @@
-     
+
   Hi there, **[[user-name]]**! Nice to meet you.
-    
+
   In the REPL you are getting back the evaluation of the expression that you typed.
     As you can see, `:user-name` is in a special form; it's called a *keyword*.
-    You have to prepend `:` to a word to create one. 
-     
+    You have to prepend `:` to a word to create one.
+
   Use the keyword `:next` to continue."
     :test #(= % :next)}
    ;; Exercise - 01
    {:title "A function for everything"
     :content
     "Let's tweak our interface!
-     
+
   You already know how to invoke functions, how to use keywords and strings.
      What if I tell you that you can change the prompt?
 
   You can call functions with keywords parameters, as `(create-dog :name \"Zeus\" :breed \"Beagle\")`
-     
-  Use the function `(set-prompt)` and set a color. It accepts optionally `:color` and `:text` as strings. 
+
+  Use the function `(set-prompt)` and set a color. It accepts optionally `:color` and `:text` as strings.
      Or click on `(set-prompt :color \"red\")`"
     :test #(contains? % :prompt-color)}
-   ;; 
+   ;;
    {:title "Functional practicioner!"
     :content
-    "Congratulations, you have called a function and changed the state of the application! 
+    "Congratulations, you have called a function and changed the state of the application!
      And the entire command was… a list!
-                                                                                     
+
   Clojure offers multiple functions to work with lists, such as `reverse`. It reverses a collection.
      So if you pass a string, it will use it as a collection of characters.
-     
+
   Type `(reverse \"a-long-string\")` to advance at the next step."
     :test #(= % (reverse "a-long-string"))}
    ;; Vectors
    {:title "We have vectors"
     :content
     "> Lisp is worth learning for the profound enlightenment experience you will have when you finally get it. - Eric Raymond
-     
+
   **Vectors** (aka arrays) contain sequential elements and they have a faster access compared to lists.
-     
+
   To create a vector you need to include the items into squared brackets `[]` without any separator.
-     
+
   Create a vector of elements, like your favorite names for cats `[\"luna\" \"milu\" \"boris\"]`. "
     :test #(vector? %)}
    ;; Variables
    {:title "Def your variables"
     :content
     "> Good programmers don't just write programs. They build a working vocabulary. - Guy Steele
-                                                                                     
+
   **Global** variables are defined using `def`. Their value could be anything.
-     
+
   Create a global variable called `foo` with a value. E.g. `(def foo \"bar\")`"
-    :test #(and (instance? SciVar %) (= "foo" (-> (.-meta %) :name str)))}
+    :test #(and (instance? IVar %) (= "foo" (-> (.-meta %) :name str)))}
```

Comment:
Here the test should now check for Var instead `(instance? Var %)`.

Reply:
@eliascotto Thank you for the correction - updated.